File: MyDailyBeat/MyDailyBeat/EVCTextFieldTableViewCell.m
//
//  EVCTextFieldTableViewCell.m
//  MyDailyBeat
//
//  Created by Virinchi Balabhadrapatruni on 10/18/14.
//  Copyright (c) 2014 eVerveCorp. All rights reserved.
//

#import "EVCTextFieldTableViewCell.h"

@implementation EVCTextFieldTableViewCell
@synthesize textField;

- (id) initWithPlaceHolder:(NSString *)placeHolder {
    NSArray *nib = [[NSBundle mainBundle] loadNibNamed:@"EVCTextFieldTableViewCell_iPhone" owner:self options:nil];
    self = [nib objectAtIndex:0];
    if (self) {
        self.placeHolderText = placeHolder;
        textField.placeholder = self.placeHolderText;
    }
    return self;
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: MDB_Swift/MyDailyBeat/MyDailyBeat/src/EVCTextFieldTableViewCell.swift

//
//  EVCTextFieldTableViewCell.swift
//  MyDailyBeat
//
//  Created by Virinchi Balabhadrapatruni on 10/18/14.
//  Copyright (c) 2014 eVerveCorp. All rights reserved.
//
import UIKit
class EVCTextFieldTableViewCell: UITableViewCell {
    @IBOutlet var textField: UITextField!
    var placeHolderText: String = ""


    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)
        // Configure the view for the selected state
    }
}
